Tests/UnitTests/Core/Propagator/AtlasStepperTests.cpp

```diff
@@ -52,11 +53,11 @@ static constexpr NavigationDirection navDir = backward;
 static const ConstantBField magneticField(Vector3D(0.1_T, -0.2_T, 2_T));
 
 // initial parameter state
-static const Vector3D pos(1_mm, -1_mm, 2_mm);
-static constexpr auto time = 2_ns;
+static const Vector4D pos4(1_mm, -1_mm, 2_mm, 2_ns);
+static const Vector3D pos = pos4.segment<3>(ePos0);
```

Comment:
Just for my understanding, what's the difference between `.segment<int DIM>` and `.block<int ROWS, int COLS>` - is the second just restricted for 1-row and or 1-col types?

Reply:
`.segment<...>()` only exists for vectors, i.e. 1d matrices while `.block<..., ...>()` can always be called.

Reply:
Thanks. As it requires one less template parameter - might that be slightly better in term of compilation resources?

Reply:
I would be surprised if that is the case. The underlying matrix type still has two dimension parameters; one is just 1.